core/templates/domain/exploration/InteractionObjectFactory.ts

```diff
@@ -184,8 +188,8 @@ export class Interaction {
    */
   static getCustomizationArgContentIds(
       customizationArgs: InteractionCustomizationArgs
-  ): string[] {
-    const contentIds = [];
+  ): (string | null)[] {
+    const contentIds: (string | null)[] = [];
```

Comment:
Update/Add a comment explaining why the return type and `contentIds` can be null ?

Reply:
added a comment